bridges/ItakuUserBridge.php

```diff
@@ -0,0 +1,1006 @@
+
+        switch ($record['action_type']) {
+                //NOTIFICATIONS WILL NOT GIVE YOU METADATA / CONTENT_OBJECT LIKE A SEARCH DOES. YOU MUST FETCH MORE DETAILS.
+            case 'liked': {
+                    if ($opt['stars']) {
+                        if (array_key_exists('comment_host_content_type', $record) && $record['comment_host_content_type'] === 'galleryimage') {
+                            $record['comment_host_content_type'] = 'image';
+                        }
+
+                        //user starred your comment on: image/post/commission title
+                        //user starred your image/post/commission: title
+                        $record['action_type'] = 'starred';
+                        $title .=  " starred your {$record['content_type']}";
+                        if ($record['content_type'] === 'comment') {
+                            $title .= ' on ' . $record['comment_host_content_type'];
+                        }
+
+                        $title .= ': ';
+
+                        $a = $this->getNotificationContent($record['content_type'], $record['object_id'], $record);
+                        $title .= $a['title'];
+                    } else {
+                        return null;
+                    }
+                    break;
+                }
+            case 'reshared': {
+                    if ($opt['stars']) {
+                        //user reshared your post/image/comment; title
+                        $title .=  " reshared your {$record['content_type']}: ";
+                        $title .= $this->getNotificationContent($record['content_type'], $record['object_id'])['title'];
+                    } else {
+                        return null;
+                    }
+                    break;
+                }
+            case 'commented': {
+                    if ($opt['comments']) {
+                        $notif_data = $this->getNotificationContent($record['content_type'], $record['object_id'], $record);
+                        if ($record['content_type'] === 'comment') {
+                            //user (replied to your comment)(commented) on commission/image/post: {title}
+                            if ($record['comment_host_content_type'] === 'galleryimage') {
+                                $record['comment_host_content_type'] = 'image';
+                            }
+                            $title .=  " replied to your comment on {$record['comment_host_content_type']}: ";
+                        } else {
+                            if ($record['content_type'] === 'galleryimage') {
+                                $record['content_type'] = 'image';
+                            }
+                            $title .=  " commented on your {$record['content_type']}: ";
+                        }
+
+                        $title .= $notif_data['title'];
+                        $content = $this->getComment($record['object_id']) . $notif_data['content'];
```

Comment:
I had a closer look and found the issue again - `$record['object_id']` is actually referencing the _parent_ comment, and not comment that is replying to the parent.

The only data that hints the actual comment is `$record['date_added']`, but even that is off by a couple of milliseconds between the time it took for the servers to register the comment, and to send out the reply notification.

In order to find the actual comment that is replying to the parent comment, I'd have to search all the comments in a post to find the closest comment posted by its `date_added` time - which does not sound like a good idea to me.

Reply:
![image](https://github.com/RSS-Bridge/rss-bridge/assets/19784752/543c16ef-9ace-4045-84e6-4f59e02457f0)